src/providers/ethereum/Transfer.ts

```diff
@@ -36,7 +42,17 @@ class Provider implements TransferProvider<ChainId, Address, Hash> {
     }
 
     async waitForTransaction(hash: Hash): Promise<void> {
-        await waitForTransactionReceipt(config, { hash, chainId: EthereumNetwork.getChainId() });
+        try {
+            await waitForTransactionReceipt(config, { hash, chainId: EthereumNetwork.getChainId() });
+        } catch (error) {
+            const blocks = await getTransactionConfirmations(config, {
+                hash,
+                chainId: EthereumNetwork.getChainId(),
+            });
+            if (blocks < 1) {
+                throw error;
+            }
+        }
```

Comment:
I think there are some mechanisms behind `waitForTransactionReceipt` and `getTransactionConfirmations` are the same.

Try increasing the `confirmations` to wait.
```suggestion
            await waitForTransactionReceipt(config, { hash, chainId: EthereumNetwork.getChainId(), confirmations: 3 });
```